### PR TITLE
Newsletter switch

### PR DESCRIPTION
**Pull request by octavioamu**
##### Description

image

##### Refers/Fixes

#4413

### CODE REVIEW

**Comment by octavioamu** [comment]
waiting new forms links

File: app/assets/v2/css/buttons.css [hunk 1]
```diff
@@ -264,3 +264,12 @@ button:focus {
   color: #666666;
   font-weight: 600;
 }
+
+.btn-gc-grey {
+  color: #171717;
+  background-color: #cbcbcb;
+}
+.btn-gc-grey:not(:disabled):not(.disabled).active {
```
Comment (thelostone-mc) [inline app/assets/v2/css/buttons.css]: add empty line?

File: app/dashboard/templates/shared/newsletter.html [hunk 2]
```diff
@@ -16,29 +16,60 @@
...
-  <form action="//gitcoin.us16.list-manage.com/subscribe/post?u=f9f93e162a8732673b942cbda&amp;id=791d905ad8"
-    method="post" id="mc-embedded-subscribe-form" class="col-12 validate"
-    name="mc-embedded-subscribe-form" target="_blank" novalidate>
-
+<div class="row newsletter text-center" id="newsletterGroup">
+  <div class="col-12">
     <h4 class="col-12 font-title font-weight-semibold py-1" id="newsletter-title">
       {% if newsletter_headline %}
-        {{ newsletter_headline }}
+      {{ newsletter_headline }}
```
Comment (thelostone-mc) [inline app/dashboard/templates/shared/newsletter.html]: could we add that indent back in ?

File: app/dashboard/templates/shared/newsletter.html [hunk 3]
```diff
@@ -16,29 +16,60 @@
...
       {% else %}
-        {% trans "Join Our Newsletter!" %}
+      {% trans "Join Our Newsletter!" %}
```
Comment (thelostone-mc) [inline app/dashboard/templates/shared/newsletter.html]: could we add that indent back in ?

File: app/dashboard/templates/shared/newsletter.html [hunk 4]
```diff
@@ -16,29 +16,60 @@
...
-      <input name="EMAIL" id="newsletter-input" placeholder="Enter your email" required>
-      <button id="newsletter-subscribe" type="submit" class="button button--primary subscribe ml-lg-2"><i class="far fa-envelope mr-2"></i>Subscribe</button>
+    I want to
+    <nav class="nav justify-content-center d-inline-flex btn-group btn-group-sm my-3" id="myTab" role="tablist">
+      <!-- <li class="nav-item"> -->
+        <a class="nav-link active btn btn-gc-grey" id="funderForm-tab" data-toggle="tab" href="#funderForm" role="tab" aria-controls="funderForm" aria-selected="true">fund bounties</a>
+      <!-- </li> -->
+      <!-- <li class="nav-item"> -->
+        <a class="nav-link btn btn-gc-grey" id="hunterForm-tab" data-toggle="tab" href="#hunterForm" role="tab" aria-controls="hunterForm" aria-selected="false">work on bounties</a>
+      <!-- </li> -->
```
Comment (thelostone-mc) [inline app/dashboard/templates/shared/newsletter.html]: remove the link tags ?

**Comment by octavioamu** [comment]
@gitcoinco/engineers Ready to review!!

**Comment by danlipert** [comment]
@octavioamu looks good to me - how was it tested?

**Comment by thelostone-mc** [comment]
^ @danlipert testing it is as simple as clicking on the tab and ensuring the subscribe button open the right form I believe!

cc @octavioamu

**Comment by octavioamu** [comment]
@thelostone-mc remember always to hard reload the browser to see the changes, here is a video  https://secure.vidyard.com/organizations/14217/players/1xuruC2LraV2ghEM482m2d?edit=true

**Comment by octavioamu** [comment]
@thelostone-mc I had added validation

**Comment by danlipert** [comment]
@thelostone-mc @octavioamu I ask just because the PR description didn't have any testing info - its a good way for us to double check as well as see what assumptions the PR creator is making during testing. Remember to hold off on merging until Aditya and I both review.

**Pull request merged**